draft-ietf-mls-protocol.md

```diff
@@ -2657,8 +2681,10 @@ itself) and S the other child:
 * The `parent_hash` field of D is equal to the parent hash of P with copath
   child S.
 
-* D is in the resolution of C, and the intersection of P's `unmerged_leaves`
-  with the subtree under C is equal to the resolution of C with D removed.
+* The last update epoch of D is equal to the last update epoch of P
+
+* The nodes between D and P are blank, and the resolution of nodes on the copath
+  from D to P are a subset of unmerged leaves of P
```

Comment:
I'm not sure to understand.
You mean that we also need the converse:
if a leaf L is in the subtree of one of the copath nodes (let's call this node N) and L is unmerged for P
then L is in the resolution of of N?

This is implied by the condition I wrote: since the resolution of N only contains leaves (that are unmerged for P), all parent nodes in N's subtree are blank. Hence every leaf in N subtree is in N resolution.